requirements.txt

```diff
@@ -6,3 +6,4 @@ bs4==0.0.1
 defusedxml==0.4.1
 lxml==3.5.0
 Unidecode==0.4.19
+pybrowserstack-screenshots==0.1
```

Comment:
This adds yet another dependency... It makes me wonder whether this should be a standalone script merged into gftools (Google Fonts Tools) instead of Fontbakery.

Even though it is not declared as a test on `Lib/fontbakery/specifications/googlefonts.py`, it still makes sense as a quality-assurance testing tool, so I think it may be indeed good to have it in the Fontbakery repo/package.

If you agree, then it seems that we'll indeed have to add this yet another dependency. And if so, then please add a setup.py entry for this new browserstack dependency.